neverwinter/nwscript/native/scriptcompfinalcode.cpp

```diff
@@ -2357,8 +2357,6 @@ int32_t CScriptCompiler::InVisitGenerateCode(CScriptParseTreeNode *pNode)
 			}
 
 		}
-
```

Comment:
Minor nitpick from me, is it normal to mix 'whitespace' fixes with working production code?

I'd consider submitting white space fixes as separate patch to reduce reviewer noise.

Reply:
All white space fixes removed in 5445009.